include/deal.II/fe/mapping_q_cache.h

```diff
@@ -111,6 +125,40 @@ class MappingQCache : public MappingQGeneric<dim, spacedim>
                const typename Triangulation<dim, spacedim>::cell_iterator &)>
                &compute_points_on_cell);
 
+  /**
+   * Initialize the data cache by computing the mapping support points for all
+   * cells (on all levels) of the given triangulation and a given @p mapping
+   * and transforming these points via the function @p transformation_function.
+   *
+   * The bool @p function_describes_relative_displacement is in the default
+   * case false and indicates that the function @p transformation_function
+   * maps to absolute coordinates.If the parameter is set to true, the return
+   * value of the function is interpreted as relative deformation and the result
+   * is added to the original point.
+   *
+   * This function calls the above function so that the comments there regarding
+   * threading are also true here.
+   *
+   * @note The cache is invalidated upon the signal
+   * Triangulation::Signals::any_change of the underlying triangulation.
+   */
+  void
+  initialize(const Mapping<dim, spacedim> &      mapping,
+             const Triangulation<dim, spacedim> &tria,
+             const std::function<Point<spacedim>(
+               const typename Triangulation<dim, spacedim>::cell_iterator &,
+               const Point<spacedim> &)> &       transformation_function,
+             const bool function_describes_relative_displacement = false);
```

Comment:
I guess you insist to have the default value false since you already changed the name? Then I would suggest to remove the default value. Default values also have the disadvantage that when extending the interface later, additional parameter must have default values as well., and that default values are error prone.